src/manifest/manifest.ts

```diff
@@ -85,6 +85,7 @@ interface IManifestParsingOptions {
   /* eslint-enable import/no-deprecated */
   /** External callback peforming an automatic filtering of wanted Representations. */
   representationFilter? : IRepresentationFilter;
+  manifestUpdateUrl? : string;
```

Comment:
Maybe we should add a comment also there.
Also we may want to specify what will happen if this is `undefined`